### ISSUE TITLE
Doesn't get song info sometimes

### ISSUE THREAD

**Issue by mchamsteryt2** [Yaron2334/Media-Control#4]
It doesn't get song info sometimes so it usually says "no song is playing" or just says ... in droptop

**Comment by paulsybrandy** [comment]
Yeah same here. It works with the Spotify app but with the Spotify app for Droptop it does not seem to realize it is playing there. It does not work with Winamp either which is surprising because that has been a long time favorite music app for many of us.
